generator/templates/rust/model.mustache

```diff
@@ -7,13 +7,20 @@
 #[serde(rename_all = "camelCase")]
 #[non_exhaustive]
 pub struct {{Name}} {
-    {{#Fields}}
+    {{#BasicFields}}
 
     {{#DocLines}}
     /// {{{.}}}
     {{/DocLines}}
     pub {{NameToSnake}}: {{{FieldType}}},
-    {{/Fields}}
+    {{/BasicFields}}
+    {{#ExplicitOneOfs}}
+
+    {{#DocLines}}
+    /// {{{.}}}
+    {{/DocLines}}
+    pub {{NameToSnake}}: Option<{{{FieldType}}}>,
```

Comment:
I have a pending PR (#96) that would (I think) simplify this kind of code.  IMHO, the `genclient.Field` should always have `Optional: true` for message and oneof fields.  I can do the cleanup after this PR or we can do it in the opposite order, your call.